client/client.go

```diff
@@ -30,6 +30,7 @@ type Client struct {
 	BaseURL    string
 	Edge       string
 	Region     string
+	AccountSid string
```

Comment:
I'm not seeing where this `AccountSid` is being used in building the v2010 path.

Reply:
It is used in api_default.go. Example [here](https://github.com/twilio/twilio-go/blob/subaccount-support/twilio/rest/api/v2010/api_default.go#L113)

Reply:
I see it now. Lost in all the docs changes. Can follow up on the generator PR.